dbt/compilation.py

```diff
@@ -238,16 +239,27 @@ def get_compiler_context(self, model, flat_graph):
         context['adapter'] = wrapper
         context['flags'] = dbt.flags
 
-        context.update(wrapper.get_context_functions())
-
         context['run_started_at'] = dbt.tracking.active_user.run_started_at
         context['invocation_id'] = dbt.tracking.active_user.invocation_id
         context['sql_now'] = adapter.date_function()
+        context['model'] = model
+        context['execute'] = True
+        context['schema'] = schema
+        context['sql'] = model['injected_sql']
+        context['log'] = logger.debug
```

Comment:
this is awesome and i can think of dozens of other things we'll want to include here in the future. A+